core/granite/commons/versions.ts

```diff
@@ -0,0 +1,94 @@
+export const MIN_OLLAMA_VERSION = "0.6.5";
+
+export function checkMinimumServerVersion(version: string): boolean {
+  const result = compareVersions(version, MIN_OLLAMA_VERSION) >= 0;
+  return result;
+}
+
+// copied from packages/config-yaml/src/schemas/data/index.ts
```

Comment:
Maybe "Regex copied from" - it's unclear what the scope of this comment applies to.